lib/collection/src/shards/telemetry.rs

```diff
@@ -21,6 +21,10 @@ pub struct ReplicaSetTelemetry {
     pub remote: Vec<RemoteShardTelemetry>,
     #[anonymize(with = anonymize_collection_with_u64_hashable_key)]
     pub replicate_states: HashMap<PeerId, ReplicaState>,
+    #[anonymize(false)]
+    pub ongoing_create_partial_snapshot_requests: usize,
```

Comment:
It should be optional, so that new version of generated client would still be capable of processing response of the old version where those fields are missing.

Reply:
Noted. Will fix.

Reply:
Done.